contracts/staking/PoSValidatorManager.sol

```diff
@@ -60,6 +68,8 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
 
     uint8 public constant MAXIMUM_STAKE_MULTIPLIER_LIMIT = 10;
 
+    uint8 public constant UPTIME_REWARDS_THRESHOLD_PERCENTAGE = 80;
```

Comment:
Lets move this to the example rewards calculator, and add the `uptimeSeconds` to the calculate rewards interface.

That allows for different rewards calculators to have different uptime requirements, or even to pro-rated rewards based on uptime.